owncloudApp/src/main/java/com/owncloud/android/presentation/providers/sharing/UsersAndGroupsSearchProvider.kt

```diff
@@ -255,15 +264,17 @@ class UsersAndGroupsSearchProvider : ContentProvider() {
     }
 
     /**
-     * Show error message
+     * Show error genericErrorMessage
      *
      * @param resource Resource with the failure information.
      */
-    private fun showErrorMessage(errorMessage: String?) {
+    private fun showErrorMessage(genericErrorMessage: String, throwable: Throwable?) {
+        val errorMessage = throwable?.parseError(genericErrorMessage, context!!.resources) ?: return
```

Comment:
here message can be nullable but above not. Can o cannot it?

Reply:
Where can it be nullable?

Reply:
throwable?.parseError(genericErrorMessage, context!!.resources) is not nullable anymore.

you can remove elvis operator and return statement 😉